packages/embeds/embed-snippet/package.json

```diff
@@ -26,7 +26,7 @@
   "types": "./dist/index.d.ts",
   "devDependencies": {
     "typescript": "^4.9.4",
-    "vite": "^4.1.2"
```

Comment:
It's downgrading vite. We should infact switch to vite 4.1.2 everywhere.

Reply:
odd